src/clr-addons/treetable/renderer/main-renderer.ts

```diff
@@ -22,6 +22,10 @@ export class TreetableMainRenderer<T = any> implements AfterViewInit {
 
   ngAfterViewInit(): void {
```

Comment:
this needs to be ngAfterViewChecked, as the rows could change during component lifetime, so new rows also get the classes